spec/src/main/asciidoc/core/injectionandresolution_full.asciidoc

```diff
@@ -0,0 +1,128 @@
+[[injection_and_resolution_full]]
+
+== Dependency injection and lookup in {cdi_full}
+
+
+[[selection_full]]
+
+=== Modularity in {cdi_full}
```

Comment:
IMO a huge part of the text from the original `Modularity` section should be here - we don't want to talk about explicit bean archive or alternatives selected per archive in Lite. WDYT?